walk-in-interview/pom.xml

```diff
@@ -67,6 +67,8 @@
                 <artifactId>appengine-maven-plugin</artifactId>
                 <version>2.2.0</version>
                 <configuration>
+                    <!--Only for selenium ui test-->
+                    <!--<port>3000</port>-->
```

Comment:
remove commented code

Reply:
The commented code is an instruction for selenium ui test, so i think we don't need to remove it